documentation/tutorials/creating_synthetic_tokens_from_truffle_console.md

```diff
@@ -33,7 +33,7 @@ This command should return “undefined”.
 const empCreator = await ExpiringMultiPartyCreator.deployed()
 ```
  
-4. Define the parameters for the synthetic tokens you would like to create. 
+4. Define the parameters for the synthetic tokens you would like to create. Note that in this example, priceFeedIdentifier, syntheticName, and syntheticSymbol are set to "UMATEST", "Test UMA Token", and "UMATEST", respectively, but you can set these parameters to any names you prefer.
```

Comment:
We can even encode this because this is markdown :)
```suggestion
4. Define the parameters for the synthetic tokens you would like to create. Note that in this example, `priceFeedIdentifier`, `syntheticName`, and `syntheticSymbol` are set to "UMATEST", "Test UMA Token", and "UMATEST", respectively, but you can set these parameters to any names you prefer.
```

Reply:
```suggestion
4. Define the parameters for the synthetic tokens you would like to create. 

Note that in this example `priceFeedIdentifier`, `syntheticName`, and `syntheticSymbol` are set to "UMATEST", "Test UMA Token", and "UMATEST", respectively, but you can set these parameters to any names you prefer in the testnet environment. <!-- TODO: add link to process for adding identifiers to mainnet when that doc is ready -->
```

Reply:
I added a bit more spacing and also a TODO to remind us to add a link describing how this will happen in mainnet.